src/string.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Equip\String;
+
+/**
+ * Convert a snake_case string to a camelCase one.
+ *
+ * @param string $snake_string The original snake_case string.
+ * @param bool $first Whether the first letter should be capitalized
+ *
+ * @return string The converted string.
+ */
+function snakeToCamelCase($snake_string, $first = false)
+{
+    $camel = implode('', array_map(static function($piece) {
+        return empty($piece) ? '' : ucfirst(strtolower($piece));
+    }, explode('_', $snake_string)));
```

Comment:
Replacing this explode with `preg_split('/_++/', $snake_string)` would also remove the need for the ternary, as empty values would be impossible.